pkg/machines/components/deleteDialog.jsx

```diff
@@ -139,11 +139,34 @@ export class DeleteDialog extends React.Component {
 
     render() {
         const id = vmId(this.props.vm.name);
+
+        let deleteButton = (
+            <Button id={`${id}-delete`} bsStyle='danger' onClick={this.open}>
+                {_("Delete")}
+            </Button>
+        );
+
+        if (!this.props.vm.persistent) {
+            deleteButton = (
+                <OverlayTrigger overlay={
+                    <Tooltip id={`${id}-delete-tooltip`}>
+                        {_("Non-persistent VM cannot be deleted. It ceases to exists when it's deactivated.")}
```

Comment:
@garrett already mentioned that he didn't like this wording. (I know you used the same in Networks and Storages) but I think we should simplify it there ans here as well.
Generally we need more direct text providing the action that needs to be done to achieve what the user wants.
Passive voice should be avoided in general in tooltip texts. Also 'deactivate' for VMs is not really used verb.
I would go with something like ''This VM is transient. Shut it down if you wish to delete it." but I would ask here for @andreasn help.

Reply:
Done